templates/common/_base/files/configure-ovs-network.yaml

```diff
@@ -302,4 +303,8 @@ contents:
       rm -f /etc/NetworkManager/system-connections/{br-ex,ovs-if-br-ex,ovs-port-br-ex,ovs-if-phys0,ovs-port-phys0}.nmconnection
       # remove bridges created by ovn-kubernetes, try to delete br-ex again in case NM fail to talk to ovsdb
       ovs-vsctl --timeout=30 --if-exists del-br br-int -- --if-exists del-br br-local -- --if-exists del-br br-ex
+
+      if [ "$iface" != "" ]; then
```

Comment:
maybe `if [ -n "$iface" ]; then`

but really we should be using `[[` with bash.

Reply:
Fixed.